index.html

```diff
@@ -24,7 +24,7 @@
             <p id="img-caption">
```

Comment:
#It would be better to use class tag instead of id tag,
.If you use id instead of class there may be a need to repeat the same CSS rule for two different elements (different ids). Code repetition is considered a bad practice and should be avoided, if possible at the design level, hence the suggestion to use class instead of id.